### PR TITLE
Added a Markdown file with a short bio.

### PR DESCRIPTION
**Pull request by vickipetrova**
PCW for Session 3.

### CODE REVIEW

**Review by willngn — APPROVED** [review]
LGTM!

File: 2022/fall/Victoria-Petrova.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Welcome to the Life of Vicki Petrova!
+
+Hi! I'm Vicki from **Bulgaria**. I love:
+
```
Comment (rebe121314) [inline 2022/fall/Victoria-Petrova.md:4]: Looks good! ::sparkle: Maybe add a few more formats, like italization.

File: 2022/fall/Victoria-Petrova.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Welcome to the Life of Vicki Petrova!
```
Comment (rebe121314) [inline 2022/fall/Victoria-Petrova.md:1]: 2022/fall/Scheffler-TT@10am/[Victoria-Petrova.md]

**Review by rebe121314 — APPROVED** [review]
✨ 👍 Great!  LGTM

**Review by rebe121314 — CHANGES_REQUESTED** [review]
In the other comments (created for consistency)

**Review by rebe121314 — APPROVED** [review]

**Pull request merged**